editor/doc/doc_data.h

```diff
@@ -74,6 +74,7 @@ class DocData {
 		String description;
 		String setter, getter;
 		String default_value;
+		bool overriden;
```

Comment:
Typo, `overridden`. (And yeah, that applies to the whole PR :P)

Reply:
The indicative present form is "override" though, only the past participle has double d.